internal/infrastructure/kubernetes/proxy/resource_provider.go

```diff
@@ -432,7 +432,7 @@ func (r *ResourceRender) PodDisruptionBudgetSpec() (*egv1a1.KubernetesPodDisrupt
 	}
 
 	podDisruptionBudget := provider.GetEnvoyProxyKubeProvider().EnvoyPDB
-	if podDisruptionBudget == nil || podDisruptionBudget.MinAvailable == nil {
+	if podDisruptionBudget == nil || podDisruptionBudget.MinAvailable == nil && podDisruptionBudget.MaxUnavailable == nil && podDisruptionBudget.Patch == nil {
```

Comment:
I assume this was a bug but can revert if this was intentional. Previously if a user _only_ wanted to provide a patch they'd also have to include `minAvailable`.